frontend/src/components/WrapTableRow.tsx

```diff
@@ -47,6 +49,24 @@ const SpacerRow = () => (
   </MUITableRow>
 )
 
+const BurnIcon = ({ color }: { color?: string }) => {
+  if (!color) {
+    color = '#CD5B5B'
+  }
+  return (
+    <LocalFireDepartmentIcon sx={{ color: {color} }} />
```

Comment:
you can do `<LocalFireDepartmentIcon sx={{ color }} />`

Reply:
tbh I'm surprised this works, becuse you're providing an object to the color property of the sx object, which should be a string, not an obect.